src/chrome/content/rules/Ghidra-SRE.org.xml

```diff
@@ -0,0 +1,9 @@
+<!--
+	For other National Security Agency coverage, see National-Security-Agency.xml
```

Comment:
Can you also add a link to this ruleset in `National-Security-Agency.xml` please?

Reply:
@Bisaloo Done.